### ISSUE TITLE
log???

### ISSUE THREAD

**Issue by lzq2369** [Xharlie/DISN#5]
nohup: ignoring input
/home/cvpruser/zsf/luzhiqing/3D/DISN/models/models
pid: 16227
Namespace(alpha=False, augcolorback=False, augcolorfore=False, backcolorwhite=False, batch_size=1, binary=False, cam_est=True, cam_log_dir='cam_est/checkpoint/cam_DISN', cat_limit=168000, category='all', create_obj=False, decay_rate=0.9, decay_step=200000, gpu='0', img_feat_onestream=False, img_feat_twostream=True, img_h=137, img_w=137, iso=0.0, learning_rate=0.0001, log_dir='checkpoint/SDF_DISN', loss_mode='3D', max_epoch=1, multi_view=False, num_classes=1024, num_points=1, num_sample_points=1, rot=False, sdf_res=256, shift=False, store=False, tanh=False, test_lst_dir='./data/filelists', threedcnn=False, view_num=24)
RESULT_OBJ_PATH:  ./demo/
HOSTNAME: o
checkpoint/SDF_DISN
here we use our cam est network to estimate cam parameters:
Tensor("Placeholder:0", shape=(), dtype=bool, device=/device:GPU:0)
WARNING:tensorflow:From /home/cvpruser/zsf/luzhiqing/luFF/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
--- cam Get model_cam and loss
odict_keys(['vgg_16/conv1/conv1_1', 'vgg_16/conv1/conv1_2', 'vgg_16/pool1', 'vgg_16/conv2/conv2_1', 'vgg_16/conv2/conv2_2', 'vgg_16/pool2', 'vgg_16/conv3/conv3_1', 'vgg_16/conv3/conv3_2', 'vgg_16/conv3/conv3_3', 'vgg_16/pool3', 'vgg_16/conv4/conv4_1', 'vgg_16/conv4/conv4_2', 'vgg_16/conv4/conv4_3', 'vgg_16/pool4', 'vgg_16/conv5/conv5_1', 'vgg_16/conv5/conv5_2', 'vgg_16/conv5/conv5_3', 'vgg_16/pool5', 'vgg_16/fc6', 'vgg_16/fc7', 'vgg_16/fc8'])
x (1, 3) y (1, 3) z (1, 3)
matrix (1, 3, 3)
trans_mat (1, 4, 3)
homo_pc.get_shape() (1, 1, 4)
pc_xyz.get_shape() (1, 1, 3)
homo_pc.get_shape() (1, 1, 4)
pc_xyz.get_shape() (1, 1, 3)
gt_xy, pred_xy (1, 1, 2) (1, 1, 2)
WARNING:tensorflow:From /home/cvpruser/zsf/luzhiqing/3D/DISN/cam_est/model_cam.py:232: get_regularization_losses (from tensorflow.contrib.losses.python.losses.loss_ops) is deprecated and will be removed after 2016-12-30.
Instructions for updating:
Use tf.losses.get_regularization_losses instead.
--- Get training operator
2019-10-31 21:58:05.544934: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-10-31 21:58:05.604962: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2100105000 Hz
2019-10-31 21:58:05.669197: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x571e4f0 executing computations on platform Host. Devices:
2019-10-31 21:58:05.669311: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
WARNING:tensorflow:From /home/cvpruser/zsf/luzhiqing/luFF/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
model_cam loaded in file: cam_est/checkpoint/cam_DISN/latest.ckpt
pred_trans_mat_val [[[-66.20531      3.0475807   -0.37013677]
  [-15.545382   -81.84503     -0.22653782]
  [-45.68309     -1.9617825    0.24275717]
  [100.045494    99.63471      1.4153072 ]]]
Tensor("Placeholder:0", shape=(), dtype=bool)
--- Get model and loss
homo_pc.get_shape() (1, 212183, 4)
pc_xyz.get_shape() (1, 212183, 3)
point_vgg_conv1 (1, 212183, 64)
point_vgg_conv2 (1, 212183, 128)
point_vgg_conv3 (1, 212183, 256)
point_vgg_conv4 (1, 212183, 512)
point_vgg_conv5 (1, 212183, 512)
point_img_feat (1, 212183, 1, 1472)
net2 (1, 212183, 1, 512)
globalfeats_expand (1, 212183, 1, 1024)
gt_sdf (1, 212183, 1)
pred_sdf (1, 212183, 1)
WARNING:tensorflow:From /home/cvpruser/zsf/luzhiqing/3D/DISN/models/model_normalization.py:285: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Model loaded in file: checkpoint/SDF_DISN/latest.ckpt
2019-10-31 21:58:12.730879
all_pts (80, 1, 212183, 3)

I want to know 
when  I run : nohup python -u demo/demo.py --cam_est --log_dir checkpoint/SDF_DISN --cam_log_dir cam_est/checkpoint/cam_DISN --img_feat_twostream --sdf_res 256 &> log/create_sdf.log &
but,the result is below:
[2] 16227
Could you give me a answer  ,why ??? what is it?

**Comment by lzq2369** [comment]
and ,I can't get the  result.obj !

**Comment by Xharlie** [comment]
hi, please google nohup command

**Comment by bharadwajdhornala** [comment]
Hi @lzq2369 Did you solve the issue? Please guide me the same.
Thanks

**Comment by judeclass** [comment]
> 而且，我无法得到 result.obj！

 Did you solve the issue? Please guide me the same.
Thanks
